packages/sw-precaching/package.json

```diff
@@ -21,5 +21,9 @@
   "homepage": "https://github.com/GoogleChrome/sw-helpers/tree/master/packages/sw-precaching",
   "main": "build/sw-precaching.min.js",
   "module": "build/sw-precaching.min.mjs",
-  "jsnext:main": "build/sw-precaching.min.mjs"
+  "jsnext:main": "build/sw-precaching.min.mjs",
+  "devDependencies": {
```

Comment:
Just checking: is this for the benefit of https://github.com/GoogleChrome/sw-helpers/pull/163/files#diff-0dba01216e2ffb2625e2115ac8158c95R1, or is it used elsewhere as well?

I remember @prateekbh ran into similar issues when he needed to use cross-module dependencies in his tests, and I think he worked around that by adding in a bunch of additional bundling steps into his build process.

I'm not sure which approach is preferable, but if you or @prateekbh have strong feelings, it would be useful to weigh in and go with a consistent approach across all of the projects.